docs/src/main/asciidoc/rest-json.adoc

```diff
@@ -552,9 +552,16 @@ to solve this problem.
 
 Configuration via an application-supplied subclass of `Application` is supported, but not required.
 
+=== Only single JAX-RS application
+
+In contrast to JAX-RS (and RESTeasy) running in a standard servlet-container, Quarkus only supports the deployment of a single JAX-RS application.
+If multiple JAX-RS `Application` classes are defined, the build will fail with the message `Multiple classes have been annotated with @ApplicationPath which is currently not supported`.
+
+If multiple JAX-RS applications are defined, the property `quarkus.resteasy.ignoreApplicationClasses=true` can be used to ignore all explicit `Application` classes. This effectively merges the JAX-RS applications into one and all resources are available via the application-path as defined by `quarkus.resteasy.path` (default: `/`).
```

Comment:
> ` This effectively merges the JAX-RS applications into one`

Are you sure? AFAIK this setting will only ignore the existing Application classes, not merge them

Reply:
Merging applications in the sense that all resources are merged/collected into one application.
Quarkus collects all rest resource classes and makes them available under either the explicit application or, if no explicit application is defined, it will make all resources available under the path as defined via quarkus.resteasy.path (lets call this "implicit application").
So, if there are multiple Jax-rs application-classes and they are all ignored, then it will effectively merge them into this implicit application.
In regular JAX-RS the, the application-class' getClasses method can be overriden to specify which resource classes belong to which application; this methods is not used in Quarkus; so all resources are just collected into whatever rest application is defined; explicit or implicit.

Reply:
Yes, I understand that, but I am not sure if when we say that this property merges the JAX-RS applications into one would not confuse the reader (but maybe that's just me :wink: )

Reply:
Ok, i have been talking about "merging jax-rs applications" to much. I have rephrased it:

If multiple JAX-RS applications are defined, the property `quarkus.resteasy.ignoreApplicationClasses=true` can be used to ignore all explicit `Application` classes. This makes all resource-classes available via the application-path as defined by `quarkus.resteasy.path` (default: `/`).